<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myScore</class>
 <widget class="QWidget" name="myScore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>成绩查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findscoreButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_findcore_back">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>返回主界面</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>259</width>
     <height>162</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="2">
     <widget class="QLineEdit" name="className_Edit"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>学号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="sNo_lineEdit"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>课程名</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>开课学期</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>授课教师</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="teacher_lineEdit"/>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="classTime_box"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="Grade_lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>190</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>240</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>成绩</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
